src/index.html

```diff
@@ -1,12 +1,300 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images\logo.png"
+          alt="logotype Moyo"
+          class="logo__image"
+        >
+      </a>
+
+      <nav>
+        <ul class="nav">
+          <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
```

Comment:
Need to fix everywhere
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
```